casepro/backend/junebug.py

```diff
@@ -115,7 +175,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_at__gte=modified_after)
```

Comment:
I meant adding it, rather than replacing it -- so all identities created between `modified_after` and `modified_before`.